### DISCUSSION TITLE
GitHub Skyline CLI extension

### DISCUSSION THREAD

**Discussion by RichCoastCR** [community/community#146705] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Do you know if GitHub Skyline CLI extension is available?

**Comment by RichCoastCR** [comment]
@arcesoftware

**Reply by arcesoftware** [reply, marked as answer]
Turn your GitHub history into a masterpiece! With the new GitHub Skyline CLI extension, generate stunning 3D skylines of your contributions and print them IRL:  [https://lnkd.in/gXMf6k_g.](url)

Let’s see your #GitHubSkyline

[https://github.blog/changelog/2024-12-09-github-skyline-cli-extension/](url)

[https://github.com/github/gh-skyline](url)

**Reply by RichCoastCR** [reply]
Perfect, thanks for letting me know. I will try it.

**Comment by RichCoastCR** [comment]
Answer provided by @arcesoftware
